### ISSUE TITLE
EDC - Maintenance

### ISSUE THREAD

**Issue by lgblaumeiser** [eclipse-tractusx/sig-release#619]
## Description

As a member of the EDC developer community,

I want to have a clean code base that follows the latest development guidelines

Therefore come code hygiene measures should be taken up, to clean up the code base, like:

- [x] Extension hygiene: Getting rid of unsupported/not officially sanctioned extensions
- [x] Extension refactoring: Rewrite/redesign legacy extensions that aren't up to Tx-EDC standards and coding guide lines
- [x] Settings hygiene: All Tractus-X EDC properties should start with tx.edc. to be able to clearly distinguish them. We should go through all extensions and fix that. --> https://github.com/eclipse-tractusx/tractusx-edc/issues/1302

Initial creator: @paullatzelsperger 

@paullatzelsperger: Feel free to extend the current description, we thought, that from a planning point of view, we could summarize these three points to one planning item and create concrete workitems during the development iteration.

## Impact

None expected

## Additional information 

- [ ] I'm willing to contribute to this feature

**Comment by paullatzelsperger** [comment]
@lgblaumeiser that's fine for the time being. Ultimately all three of these items should become issues in the [tractusx-edc](https://github.com/eclipse-tractusx/tractusx-edc) repo

**Comment by stephanbcbauer** [comment]
was presented in the open planning ⇾ contributors and committer are involved.

**Comment by stephanbcbauer** [comment]
Hello @paullatzelsperger, @lgblaumeiser 

Since the feature is a 24.08 feature and the development phase is coming to an end, we need a status on the feature.

- [ ] Currently you are assigned (Responsible) → Is this correct? If not, please assign the correct contact person
- [ ] Please check whether the status (backlog, work in progress ...) is set correctly
- [ ] Please check whether there is an assignment to a committee or expert group (supported-by) to be found on the [board](https://github.com/orgs/eclipse-tractusx/projects/26/views/28)
- [ ] Please comment on the current status of the feature
- [ ] Are all SubTasks (issues from other repositories that deal with the feature) linked? → The easiest way is to mention the feature here in the issue (via the ID) so we can see which teams/repositories are involved.

If you need any clarification, please get in touch, thank you very much.
Stephan

**Comment by paullatzelsperger** [comment]
@stephanbcbauer this is a long-running issue, that is never really "done". I cannot tell you all maintenance tasks that'll ever be in advance, because we typically refactor, cleanup and maintain when we come across problems.

TL;DR: we could close this issue, and create dedicated ones whenever we do a refactoring, WDYT?

**Comment by lgblaumeiser** [comment]
2024-06-11: Onoging activity, will be closed close to release with a documentation of issues closed during the development cycle.

**Comment by lgblaumeiser** [comment]
2024-06-11:

* [x]  Currently you are assigned (Responsible) → Is this correct? If not, please assign the correct contact person
* [x]  Please check whether the status (backlog, work in progress ...) is set correctly
* [x]  Please check whether there is an assignment to a committee or expert group (supported-by) to be found on the [board](https://github.com/orgs/eclipse-tractusx/projects/26/views/28)
* [x]  Please comment on the current status of the feature:
         Ongoing activity, will be closed close to release with a documentation of issues closed during the development cycle.
* [ ]  Are all SubTasks (issues from other repositories that deal with the feature) linked? → The easiest way is to mention the feature here in the issue (via the ID) so we can see which teams/repositories are involved.
        @paullatzelsperger : Please add all activity issues and PRs connected to this activity to the item.

**Comment by lgblaumeiser** [comment]
Will be closed with the release of the Tractus-X connector

**Comment by stephanbcbauer** [comment]
Hello @lgblaumeiser 
 
In a few weeks, we want to publish Tractus-X Release 24.08. For this, we need a clear status about the features (which will be included in the release and which will be cancelled or postponed to the next release).
 
Currently, the feature is still in status `Work in progress`. Please update the status by 09.07.24 eob.
 
If the feature is not ready for release 24.08, or it is possibly not planned at all, please set the status to `Not in release`. Please remember that in this case, any dependent components must be informed.
 
Thank you and best regards.

**Comment by lgblaumeiser** [comment]
Hi @paullatzelsperger @wolf4ood @ndr-brt 

can you add links to PR which you would rate under this category, so that we can close the issue, thanks!

**Comment by wolf4ood** [comment]
adding few more:

- https://github.com/eclipse-tractusx/tractusx-edc/pull/1401
- https://github.com/eclipse-tractusx/tractusx-edc/pull/1414
- https://github.com/eclipse-tractusx/tractusx-edc/pull/1396

**Comment by ndr-brt** [comment]
- https://github.com/eclipse-tractusx/tractusx-edc/pull/1399
- https://github.com/eclipse-tractusx/tractusx-edc/pull/1402
- https://github.com/eclipse-tractusx/tractusx-edc/pull/1309
- https://github.com/eclipse-tractusx/tractusx-edc/pull/1340
